Getting-Started-With-Orleans/Running-the-Application.md

```diff
@@ -8,12 +8,11 @@ title: Running the Application
 
 To allow applications to communicate with grains from outside Orleans, the framework includes a client library. 
 This client library might be used by a desktop or mobile application, or by a front end server that renders interactive web pages or exposes a web services API. 
-The client library provides a subset of the Orleans programming model for writing asynchronous clients that can find, create, and communicate with Orleans grains. This requires a few simple steps:
+The client library provides a subset of the Orleans programming model for writing asynchronous clients that communicate with Orleans grains. This requires a few simple steps:
 
-1. Connect to an Orleans gateway 
-2. Find existing grains or create new ones 
-3. Send messages to grains and receive responses 
-4. Receive asynchronous notifications from grains via observers 
+1. Connect to an Orleans gateway  
+2. Send messages to grains and receive responses 
+3. Receive asynchronous notifications from grains via observers 
```

Comment:
Given this last "step" isn't required to work with grains, maybe this section should be rewritten to say something along the lines of:
"This requires connecting to an Orleans gateway. Once connected, a client can send messages to grains, receive responses, and receive asynchronous notifications from grains via observers."